<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QMainWindow" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compiler_Pro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPlainTextEdit" name="inputTextEdit"/>
      <widget class="QPushButton" name="compileButton">
       <property name="text">
        <string>Компилировать</string>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="LexTable">
           <column>
            <property name="text">
             <string>Лексема</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Тип</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Данные</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="rulesTextEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPlainTextEdit" name="objectCodeTextEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="optimizedCodeTextEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="lexTableAction"/>
    <addaction name="ruleAction"/>
    <addaction name="objectCodeAction"/>
    <addaction name="optimizedCodeAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Что это?</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="lexTableAction">
   <property name="text">
    <string>Таблица Лексем</string>
   </property>
  </action>
  <action name="ruleAction">
   <property name="text">
    <string>Правила синтаксического анализа</string>
   </property>
  </action>
  <action name="objectCodeAction">
   <property name="text">
    <string>Объектный код</string>
   </property>
  </action>
  <action name="optimizedCodeAction">
   <property name="text">
    <string>Оптимизированный код</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
